conans/test/functional/toolchains/gnu/test_pkgconfigdeps.py

```diff
@@ -0,0 +1,253 @@
+import os
+import platform
+import textwrap
+import unittest
+
+import pytest
+
+from conans.test.utils.tools import TestClient
+from conans.util.files import load
+
+
+class PkgGeneratorTest(unittest.TestCase):
```

Comment:
Start using new ``pytest`` style test, not unittest.

Reply:
Yeah, sure!